assets/js/modules/tagmanager/components/common/AMPContainerSelect.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Tag Manager AMP Container Select component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import ContainerSelect from './ContainerSelect';
+import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
```

Comment:
Sometimes I wonder about moving these all to a single constants file (or re-exporting them from one), so you could `import { CORE_SITE, CORE_USER, MODULE_ANALYTICS } from 'googlesitekit-data` or something. Just thinking aloud and not related to this issue, but something to ponder 🙂

Reply:
Yeah, I've been thinking about that too.

We could do something like this? I would prefer to use this in `registerAllStoresOn` too.
```js
export const STORE_NAME = 'core/site';
export { STORE_NAME as CORE_SITE }
```

Reply:
Hmm, I prefer to keep these constants separated since the store names are part of the respective stores, not so much the registry `googlesitekit-data`. _Maybe_ we can create some kind of `default-stores.js` file or so where all of these constants are imported from their actual files and then exported with these more specific names - but I also think having to write the individual imports like now is not really that bad?

Reply:
> I also think having to write the individual imports like now is not really that bad?

It's not terrible, but now that we've had to write them 1000 times it would be really nice if we could leverage auto-imports for them, and we can't do that as it is because the exports all have the same name. This would also help ensure consistency in the name that's imported though as well.

The pain for me at least isn't really the alias, it's just looking-up/finding the import which you need to do manually more/less (yes you can copy-paste from another file, but that's also somewhat manual 😉 ).